archive.php

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="archive.css">
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+</head>
+<body>
+	<?php
+	include 'navbar.php';
+	?>
+	<div id="container">
+		<script>
+		var youtube_videos = ['https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
```

Comment:
@hitesh4 You don't need to include the repetitive string `https://www.youtube.com/embed/` here. Just keep the video ID's in this array, and then add that string to the ID while generating the elements.

**

cc @students-alumni-cell-iitkgp/web-team-members @aniarya82